### DISCUSSION TITLE
Errore nel log

### DISCUSSION THREAD

**Discussion by CarloCalv** [indomus/forum#2114] [HUB: Home Assistant]
Ciao,
mi sono accorto che facendo un controllo sui registri mi da questo errore:

Logger: homeassistant.helpers.template
Source: helpers/template.py:583
First occurred: 09:47:45 (534 occurrences)
Last logged: 14:14:17

Template variable error: 'value_json' is undefined when rendering '{{ value_json.last_update }}'

si ripete ogni 30 secondi

potreste aiutarmi a identificare come risolvere il problema.

Grazie Carlo

**Comment by snakuzzo** [comment]
Innanzi tutto guarda il log di HA per intero...non solo quella sintesi che hai postato.

In teoria se si tratta di entità create a mano, hai una qualche entità il cui stato viene recuperato da un JSON...ma tale JSON di fatto non esiste.

Se invece si tratta di entità generate in automatico da integrazioni (tipo autodiscovery di zigbee2mqtt), devi vedere sulla pagina del progetto relativo.

**Reply by CarloCalv** [reply]
grazie per la pronta risposta,
il log di ha mi riporta  questo messaggio per n volte
2022-11-27 18:30:52.163 ERROR (MainThread) [homeassistant.helpers.template] Template variable error: 'value_json' is undefined when rendering '{{ value_json.last_update }}'

non ho niente zigbee o mqtt, non
come faccio a trovare il JSON che mi crea problema

**Reply by snakuzzo** [reply]
Se non hai template tuoi...metti il log a debug con logger e vedi cosa succede prima di quel messaggio

**Reply by CarloCalv** [reply]
ho diversi template miei, ma non saprei quale sia quello che mi da il problema, scusami ma non sono molto pratico come faccio a mettere il log a debug .....   Grazie per la pazienza

**Reply by snakuzzo** [reply]
[QUI](https://www.home-assistant.io/integrations/logger/) trovi la documentazione di logger.

Se sono template tuoi usa un editor decente e fai una ricerca a tappeto di `value_json.last_update`...se non è roba tua...fai come ti ho scritto prima

**Reply by CarloCalv** [reply]
ok grazie spero di risolvere ti farò sapere

**Reply by CarloCalv** [reply]
ho usato VsCode ed ho risolto il problema.

Grazie tante utilissimo e gentilissimo come sempre
